api/capi/include/nnstreamer.h

```diff
@@ -224,6 +230,18 @@ typedef void (*ml_pipeline_sink_cb) (const ml_tensors_data_h data, const ml_tens
  */
 typedef void (*ml_pipeline_state_cb) (ml_pipeline_state_e state, void *user_data);
 
+/**
+ * @brief Callback to execute the custom filter in NNStreamer pipelines.
+ * @since_tizen 6.0
+ * @remarks The @a in can be used only in the callback. To use outside, make a copy.
+ * @remarks The @a out can be used only in the callback. To use outside, make a copy.
+ * @param[out] in The handle of the tensor input (a single frame. tensor/tensors).
```

Comment:
maybe `param[in]`? because `in` is passed as `const`?

Reply:
this line defines user callback, out is right.

Reply:
Oh.. got it.